src/since.rs

```diff
@@ -103,6 +186,18 @@ impl EpochNumberWithFraction {
     pub const LENGTH_MAXIMUM_VALUE: u64 = (1u64 << Self::LENGTH_BITS);
     pub const LENGTH_MASK: u64 = (Self::LENGTH_MAXIMUM_VALUE - 1);
 
+    pub fn create(number: u64, index: u64, length: u64) -> Option<EpochNumberWithFraction> {
+        if number < Self::NUMBER_MAXIMUM_VALUE
```

Comment:
suggest to add checking for `index < length`, ref: https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0030-ensure-index-less-than-length-in-since/0030-ensure-index-less-than-length-in-since.md

Reply:
Added